site/content/docs/4.3/getting-started/theming.md

```diff
@@ -248,8 +248,20 @@ All colors available in Bootstrap 4, are available as Sass variables and a Sass
   {{< theme-colors.inline >}}
   {{- range $.Site.Data.colors }}
     {{- if (and (not (eq .name "white")) (not (eq .name "gray")) (not (eq .name "gray-dark"))) }}
-    <div class="col-md-4">
-      <div class="p-3 mb-3 swatch-{{ .name }}">{{ .name | title }}</div>
+    <div class="col-md-4 mb-3 font-monospace">
+      <div class="p-3 mb-2 swatch-{{ .name }}">
+        <strong class="d-block">${{ .name }}</strong>
+        {{ .hex }}
+      </div>
+      <div class="p-3 bd-{{ .name }}-100">${{ .name }}-100</div>
```

Comment:
This could be simplified with a `range`, ping me if I forget.

Reply:
Ping? 😄

Reply:
I'm gonna have a look soon-ish.